pyproject.toml

```
@@ -6,11 +6,16 @@ exclude = '''
 (
       \.git
     | \.venv
-    | __pycache__
-    | data
-    | docs
-    | spec
-    | build
-    | dist
+    | /__pycache__
+    | /data
+    | /docs
+    | /spec
+    | /build
+    | /dist
+    | /venv
 )
 '''
+
+[tool.pylint.format]
+max-line-length = 100
```

```suggestion
max-line-length = 120
```
Due to the black setting above